app/controllers/organization_transactions_controller.rb

```diff
@@ -12,7 +12,7 @@ def index
   end
 
   def new
-    @organization_transaction = @organization.organization_transactions.build
+    @organization_transaction = @organization.organization_transactions.build(posted_at: Date.current)
```

Comment:
Shouldn't `posted_at` be managed entirely server side when the transaction is saved? If true, this would apply to the whitelisted strong params too.

Reply:
No. Historical payments can be applied. This is especially important as we migrate data from v1.